service/api_plugin_s3copy.go

```diff
@@ -134,7 +134,8 @@ func (as *APIServer) s3copyPlugin(w http.ResponseWriter, r *http.Request) {
 		})
 
 	if err != nil {
-		grip.Error(errors.Wrap(errors.WithStack(newPushLog.UpdateStatus(model.PushLogFailed)), "updating pushlog status failed"))
+		// this is not using a constant because the code will be removed soon
```

Comment:
Can you attach a `// TODO (<ticket_num>)` for which ticket is reponsible for removing it?

Reply:
done. This whole file should be removed.